code/modules/antagonists/traitor/objectives/steal.dm

```diff
@@ -245,3 +244,115 @@ GLOBAL_DATUM_INIT(steal_item_handler, /datum/objective_item_handler, new())
 	SIGNAL_HANDLER
 	if(objective_state == OBJECTIVE_STATE_ACTIVE)
 		START_PROCESSING(SSprocessing, src)
+
+/obj/item/traitor_bug
+	name = "suspicious device"
+	desc = "It looks dangerous."
+	item_flags = EXAMINE_SKIP
+
+	icon = 'icons/obj/weapons/items_and_weapons.dmi'
+	icon_state = "bug"
+
+	/// The area at which this bug can be planted at. Has to be a type.
+	var/area/target_area_type
```

Comment:
We maintaining this behaviour in case someone else brings the objective back? Otherwise a fair amount of branching in this code can probably be cut.